documentation/assemblies/assembly-external-logging.adoc

```diff
@@ -0,0 +1,36 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration.adoc
+
+[id='external-logging_{context}']
+= External logging
+
+When setting the logging levels for a resource, you can specify them _inline_ directly in the `spec` of the resource YAML:
```

Comment:
in the `spec.logging` section of the resource? what do you think?

Reply:
:+1:  I changed to: the `spec.logging` property